src/Configuration.jl

```diff
@@ -21,6 +21,7 @@ Base.@kwdef mutable struct ServerOptions
     disable_writing_notebook_files::Bool = false
     notebook::Union{Nothing,String} = nothing
     simulated_lag::Real=0.0
+    enable_rest::Bool = false
```

Comment:
Is there a significant performance impact of having the REST API enabled?
Otherwise, I would suggest to enable it as default.

Reply:
There isn't any performance impact, but right now there are zero security features at all. It's probably not a big deal though, so I will update that